Cython/Utility/ModuleSetupCode.c

```diff
@@ -107,64 +106,9 @@
   #undef CYTHON_USE_EXC_INFO_STACK
   #define CYTHON_USE_EXC_INFO_STACK 0
 
-#elif defined(PYSTON_VERSION)
```

Comment:
I would need to change it to check for `PYSTON_MAJOR_VERSION` or something similar because we don't define `PYSTON_VERSION` right now (in Pyston v2). 
I would prefer to only add it in the future if it turns out to be necessary but if you prefer to have it in I will change my commit.